server/db/seed/Availability.js

```diff
@@ -0,0 +1,17 @@
+// // day: DataTypes.INTEGER,
```

Comment:
didn't want to add it to the db since I haven't thought seriously about how it will be used in the logic.